### PR TITLE
Update migrate113 port

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

File: nas_ports/freenas/freenas-migrate113/Makefile [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # $FreeBSD$
 
 PORTNAME=	freenas-migrate113
-PORTVERSION=	2019112901
+PORTVERSION=	0.1
```
Comment (william-gr) [inline nas_ports/freenas/freenas-migrate113/Makefile]: Why the version change?

0.1 is smaller than 2019112901 so we need PORTEPOCH set if we really need that

Comment (themylogin) [reply nas_ports/freenas/freenas-migrate113/Makefile]: My copypasted script from zettarepl port did it :)

File: nas_ports/freenas/freenas-migrate113/Makefile [hunk 2]
```diff
@@ -1,7 +1,8 @@
 # $FreeBSD$
 
 PORTNAME=	freenas-migrate113
-PORTVERSION=	2019112901
+PORTVERSION=	2020000000
+PORTREVISION=	1
```
Comment (william-gr) [inline nas_ports/freenas/freenas-migrate113/Makefile]: Dont need the revision, fwiw

Comment (themylogin) [reply nas_ports/freenas/freenas-migrate113/Makefile]: It will be the one that is incremented every time

**Pull request merged**